README.md

```diff
@@ -27,3 +27,4 @@ Pentru a menține proiectul organizat, te rugăm să urmezi aceste reguli atunci
 # Resurse
 
 [FREE-GITHUB PACK](https://education.github.com/pack) - O multitudine de platforme gratuite oferite in parteneriat cu GitHub,pentru elevi si studenti.
+[C#](https://www.w3schools.com/cs/index.php) - Introducere in C#
```

Comment:
Good resource to start with getting familiar with C#, also respected my format layout for this document, I'm waiting for more resources and even more, try to customize this document, look out for MARKDOWN tutorial